src/main/resources/default/taglib/t/helpbox.html.pasta

```diff
@@ -1,8 +1,9 @@
+<i:arg type="String" name="class" default="" description="Permits to supply additional CSS classes" />
```

Comment:
Optional: Wouldnt it generally be better to first name the name of the parameter and then the type as the param name is the first thing you need to know when calling a template and in my opinion is also more important than the param type (especially for optional params)?

Reply:
true - coming from java it was kind of the default to declare a variable "String class" and thus the pattern was chose. We can switch this around, but to make it consistent I'd do it for every template in a separate PR

Reply:
Sounds great